Fitter/TauES_ID/harvestDatacards_TES_idSF_MCStat.py

```diff
@@ -90,6 +105,7 @@ def harvest(setup, year, obs, **kwargs):
             scaleFactor = 1.0  
             if "scaleFactor" in sysDef:
               scaleFactor = sysDef["scaleFactor"]
+            if "name" in sysDef: sysDef["processes"] = check_integral(filename, sysDef["processes"], sysDef["name"], region)
```

Comment:
what's this new function for ? can you put a comment line to explain, pls?

Reply:
For some systematic, I saw that statistics was too small, actually zero, and in some cases it was giving nan value. If those samples were removed from the samples list, nan value was removed.